lighthouse/helpers/dart_db.py

```diff
@@ -23,8 +25,10 @@ def get_samples_for_barcode(cnxn, barcode):
         f"SELECT * FROM {app.config['DART_RESULT_VIEW']}"
         f" WHERE [{FIELD_DART_DESTINATION_BARCODE}]='{barcode}'"
         f" AND (([{FIELD_DART_ROOT_SAMPLE_ID}] IS NOT NULL"
+        f" AND [{FIELD_DART_SAMPLE_UUID}] IS NOT NULL"
         f" AND [{FIELD_DART_RNA_ID}] IS NOT NULL"
-        f" AND [{FIELD_DART_LAB_ID}] IS NOT NULL)"
+        f" AND [{FIELD_DART_LAB_ID}] IS NOT NULL"
+        f" AND [{FIELD_DART_SAMPLE_UUID}] IS NOT NULL)"
```

Comment:
Looks like this line is included twice. Possibly a merge conflict resolution thing we got wrong

Reply:
Removed duplicate 👍